src/corebluetooth.cs

```diff
@@ -935,19 +942,23 @@ interface CBPeer : NSCopying {
 		NSUuid Identifier { get; }
 	}
 
+	// The base type exists on macOS 32 but the fields do not
 	[Watch (4,0)][iOS (11,0)][TV (11,0)][Mac (10,13)]
 	[BaseType (typeof (NSObject), Name = "CBL2CAPChannel")]
 	interface CBL2CapChannel {
-
+		[Mac (10,13, onlyOn64: true)]
```

Comment:
move it to the type definition (it's already there without the 64 bits)

Reply:
I originally did that, but it turns out:

https://gist.github.com/chamons/1def72ee89294ae691a855f035c7c0f1

that the properties are all 64-bit only, but you can create an instance of the type in 32-bit.

Which would be useless, but there are a number of uses in this file as param to functions, and those functions are _not_ 64-bit only.

:(

I can mark this, and those functions transitively, as 64-bit only, but I was trying to match the joy that is the native SDK.

Reply:
It does not look very usable - but let's assume there's some reason we don't know for this.
Still let's add a comment on the type about this fact, like:
```
// The type is available in 32bits macOS 10.13 even if most properties are 64 bits only
```